examples/README.md

```diff
@@ -0,0 +1,94 @@
+# Examples
+
+This package contains all of the sources that appear in the Language Tour and the Library Tour, including the smallest of code excerpts.
+
+## Original sources
+
+Sources originally taken from the Up-and-Running repo (but reworked to contain doc regions)
+are in these folders:
+
+- `language_tour` (now obsolete, see below)
+- `library_tour`
+
+These original sources don't have tests, and practically each code excerpt is an independent
+package (with its own pubspec).
+
+## New sources
+
+Consolidated and reworked versions of the original sources are being developed.
+So far, the Language Tour sources have been completely rewritten, and are found
+in these folders:
+
+- `lib/language_tour`
+- `test/language_tour`
+
+As can be expected, Travis jobs run the
+
+- Analyzer over both `lib` and `test`
+- Tests under `test`
+
+### Source organisation
+
+The new Language Tour sources are under `lib/language_tour` and `test/language_tour`.
+Below each of these folders, you'll find either a file or a folder with a name
+that matches a top-level section of the Language Tour. This file/folder contains
+all of the code excerpt sources for that section.
+
+Under `lib` you might find a function defined as:
+
+```dart
+void miscDeclAnalyzedButNotTested(bool c) {
+  {
+    // #docregion foo-1
+    var foo = 1
+    // #enddocregion foo-1
+  }
+
+  {
+    // #docregion foo-2
+    var foo = 2
+    // #enddocregion foo-2
+  }
+
+  // ...
+}
+```
+
+That is, code regions are placed with a (headless) code block. This ensures
+that code region declarations (which sometimes re-declare the same entity)
+don't clash.
+
+### Where to find a code excerpt
+
+Generally speaking, a small code excerpt that is subject to testing is embedded
+directly in its test. Code regions that are subject to analysis only
+are in `lib`. Larger examples are in `lib`, and their tests (if any) under `test`.
+
+### Why not test all?
+
+Not all code excerpts are tested because some are just fragments of anything
+useful, and others illustrate features that would required significant test
+scaffolding to be written and the effort isn't worth the small gain.
+
+### _$print_
+
+All code excerpts use `$print()` instead of `print()`. The global `$print`
+variable (defined in `lib/util/print.dart`) is overwritten when running
+tests so that we can capture print output. By default `$print == print`.
```

Comment:
This might need a bit more motivation here. Maybe show a code excerpt where you're testing the input, or just link to a dart file that uses `$print` and is most likely to survive forever.

Also, should we _always_ use `$print`, even in excerpts that are not tested (yet)? If so, clarify the language — "you should always use $print because ...".